client/components/cart.jsx

```diff
@@ -18,9 +35,9 @@ class Cart extends React.Component {
               <div className='quantity-container'>
                 <span className='quantity-text'>QNTY</span>
                 <div className='quantity-buttons'>
-                  <i className="fa-solid fa-plus" />
+                  <i className="fa-solid fa-plus plus" onClick={this.plusHandler} />
                   <span className='quantity-value'>{e.quantity}</span>
-                  <i className="fa-solid fa-minus" />
+                  <i className="fa-solid fa-minus minus" onClick={this.minusHandler} />
```

Comment:
I recommend making these buttons so that double-clicking them does not highlight the page text